subgraphs/uniswap-forks/protocols/camelot-v2/src/mappings/pool.ts

```diff
@@ -0,0 +1,90 @@
+import { BigInt } from "@graphprotocol/graph-ts";
+import {
+  Burn,
+  FeePercentUpdated,
+  Mint,
+  SetStableSwap,
+  Swap,
+  Sync,
+  Transfer,
+} from "../../../../generated/templates/Pair/Pair";
+import { UsageType } from "../../../../src/common/constants";
+import {
+  updateFinancials,
+  updatePoolMetrics,
+  updateUsageMetrics,
+} from "../../../../src/common/updateMetrics";
+import { convertTokenToDecimal } from "../../../../src/common/utils/utils";
+import {
+  handleBurn as handlePoolBurn,
+  handleMint as handlePoolMint,
+  handleSync as handlePoolSync,
+  handleTransfer as handlePoolTransfer,
+} from "../../../../src/mappings/pool";
+import { createPoolFees } from "../../src/common/creators";
+import {} from "../../src/common/updateMetrics";
+import { INT_THREE, PairType } from "../common/constants";
+import { createSwapHandleVolumeAndFees } from "../common/creators";
+import { _HelperStore } from "../../../../generated/schema";
+
+// Handle transfers event.
+// The transfers are either occur as a part of the Mint or Burn event process.
+// The tokens being transferred in these events are the LP tokens from the liquidity pool that emitted this event.
+export function handleTransfer(event: Transfer): void {
```

Comment:
This is a nit pick, and you may like how you have it now better, so I will leave it up to you, but you could also just point the subgraph.yaml template for camelot to the handlers in the main /src library for `Transfer`, `Sync`, `Deposit`, and `Withdraw`, so you do not have to create an extra handler for each. I think it would also help clarify what is different here, too.

Reply:
I agree it would be better that way and that's what I tried to do first, but there was some issue with splitting the event handlers to different files, which is why I ended up doing it this way instead.